docs/journalist/faq.rst

```diff
@@ -0,0 +1,44 @@
+FAQ
+===
+
+Frequently Asked Questions
+--------------------------
+
+How secure is Qubes? How does the security of this system compare to using an air-gapped Secure Viewing Station?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+[This section is a stub]
+  - Audits
+  - grsec
+  - Qubes OS team
+  - Our security team
+  - Discuss Xen
+  - Discuss benefits of avoiding systems where security responsibility is on the user (opsec concerns)
+
+Can I install custom software on the SecureDrop Workstation?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Right now, the pilot project is designed to make the journalist experience
+easier by combining the functionality of the Journalist Workstation and Secure
+Viewing Station. The main focus is making sure that checking SecureDrop is
+easier and faster. While we hope to add advanced tooling and document-
+processing options down the line, at this time we request that you leave the
+workstation configured the way it was provisioned, and hold off on installing
+additional software or changing the functionality or configuration of any of
+the VMs. If you have specific needs that you'd like to discuss with us, open an
+issue on Redmine or send us an email at support@freedom.press.
+
+Why can’t I save or print from the Viewer VM apps?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+[This section is a stub]
+ - Sandbox/isolation and comparison to SVS
+ - Viewer VM being a "true" sandbox means avoiding opening potentially
+   malicious files in environments where you may also attach external devices
+   (printers, USB sticks); the workstation prevents this
+
+My downloads and uploads are time-consuming
```

Comment:
I don't know that we need to add this Q until we get it from users. Downloading over Tor _should_ not be slower than in the classic SecureDrop experience, though we should test reliability with large files. The user can only upload messages, which should generally be fast.

I do think we'll probably need to add information about the startup time, when connecting to a server with a lot of submissions.

Reply:
Ok - I'll amend FAQ to add that. I was aiming for addressing some of the more salient questions from #10

Reply:
see 1447bdd

Reply:
Makes sense, I especially think speaking to the delay in opening submissions is useful -- there is very limited user feedback right now (in the form of Qubes notifications), so "why does it take so long to open anything" is a good question to anticipate.